docs/autotune/setup.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Autotune
+sidebar_label: Setup
+slug: /autotune/setup
+---
+
+## How to set up Autotune
+
+1. To create a new Autotune experiment, navigate to the Autotune section on the Statsig console: https://console.statsig.com/ 
```

Comment:
To create a new Autotune experiment, navigate to the [Autotune section on the Statsig console](https://console.statsig.com/autotune).

You can link it like this :)

Reply:
😮